lib/src/components/RadioGroup/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { RadioGroup as AriakitRadioGroup, useRadioStore, useStoreState } from '@ariakit/react'
+import type { Ref } from 'react'
+import { forwardRef } from 'react'
+
+import { FieldGroup } from '@/components/FieldGroup'
+import { Radio } from '@/components/Radio'
+import { classNames } from '@/utils'
+
+import radioGroupStyles from './radio-group.module.scss'
+import type { RadioGroupOption, RadioGroupProps } from './types'
+
+const cx = classNames(radioGroupStyles)
+
+export const RadioGroup = forwardRef<HTMLFieldSetElement, RadioGroupProps>(
+  (
+    {
+      className,
+      dataTestId,
+      disabled,
+      id,
+      label,
+      name,
+      onChange,
+      options = [],
+      renderOption: Component = Radio,
+      required,
+      value,
+      ...rest
+    },
+    ref
+  ) => {
+    const store = useRadioStore({ defaultValue: value })
+    const activeValue = useStoreState(store, 'value')
+
+    const handleChange = (valueSelected: RadioGroupOption['value']) => {
+      if (disabled) return
+
+      store.setValue(valueSelected)
+      onChange?.(valueSelected)
+    }
+
+    return (
+      <FieldGroup
+        as={AriakitRadioGroup}
+        className="mb-0"
```

Comment:
If I remember correctly, this component isn’t “exposed”. Since it’s only used internally, why not move the mb-0 directly into its styles? you could also override this with a custom prop 🤔